### ISSUE TITLE
Floorplan Tab Not Appearing

### ISSUE THREAD

**Issue by ebboyer** [tbotnz/netbox_floorplan#15]
**Problem Summary:**
A user (that is not a Netbox superuser) has been assigned all available Netbox Floorplan permissions and is experiencing the following:

- Cannot view/create/edit/delete floorplans via the "Floorplan" tab in any new or existing locations.
- Can, however, create new floorplans manually through the admin panel.
- When trying to create a floor plan using the netbox-root-url/plugins/floorplan/floorplans/add URL, the GET request shows 403 forbidden

**non-superuser view:**
![tab_missing](https://github.com/tbotnz/netbox_floorplan/assets/129799095/ffb48bfa-8bae-4449-bb32-dd926f38be4d)

**superuser view of the same view:**
![Screenshot 2023-10-27 at 11 19 38 AM](https://github.com/tbotnz/netbox_floorplan/assets/129799095/bd11f664-952d-4b4c-8103-255a2386f087)

**Expected Results**
I would expect the Floorplan tab to be visible to this user and usable for all of the functions they have permissions for.

User's Environment:
OS: Ubuntu 22.04 Desktop
Broswers: Chrome & Firefox were tried

**Comment by moonrail** [comment]
If I am not mistaken, the permission required in the ViewTab is incorrect.

- Model class is named `Floorplan`: https://github.com/tbotnz/netbox_floorplan/blob/0.2/netbox_floorplan/models.py#L10
  - therefore app.model is rendered as `netbox_floorplan.floorplan` (visible via e.g. API)
  - therefore with the desired action `view` it should be: `netbox_floorplan.view_floorplan`
- ViewTabs use `floorplanobject` instead: https://github.com/tbotnz/netbox_floorplan/blob/0.2/netbox_floorplan/views.py#L20

So `netbox_floorplan.view_floorplanobject` should be `netbox_floorplan.view_floorplan`.

**Comment by tbotnz** [comment]
Feel free to open a PR

**Comment by ebboyer** [comment]
@tbotnz @moonrail Thanks, both of you! Per @moonrail's suggested changes, I created a PR reflecting the changes they suggested.

**Comment by ebboyer** [comment]
@tbotnz Thanks for merging that PR! Upon testing things after that merge, I discovered that I could view the floorplan tab now as a non-super-user, but still couldn't create new floorplans without usage of the admin panel. Here is the page that I got when I clicked the "Add Floorplan" button in a location that didn't have one associated yet

![Screenshot 2023-11-15 at 9 28 31 AM](https://github.com/tbotnz/netbox_floorplan/assets/129799095/eded400f-ab8e-4fb6-80b2-745b2a22816e)


I looked back through the code and found more changes that would need to be made that are using the same logic as what @moonrail had discovered previously with the naming of these permissions. Particularly, this line here is likely what is causing this to fail, but there's an additional line that I modified in another PR that I just put in

https://github.com/tbotnz/netbox_floorplan/blob/a65835aa5db2528da75deb05121dbdf102248cd1/netbox_floorplan/views.py#L61
